articles/azure-functions/functions-monitoring.md

```diff
@@ -174,7 +174,7 @@ The *host.json* file configures how much logging a function app sends to Applica
 
 This example sets up the following rules:
 
-1. For logs with category "Host.Results" or "Function", send only `Error` level and above to Application Insights. Logs for `Information` level and below are ignored.
+1. For logs with category "Host.Results" or "Function", send only `Error` level and above to Application Insights. Logs for `Warning` level and below are ignored.
```

Comment:
Based on the table above, the level before "Error" is "Warning", not "Information"